### ISSUE TITLE
Standardize `RData` delimiter 

### ISSUE THREAD

**Issue by mistermoe** [decentralized-identity/did-dht#83]
<img width="745" alt="image" src="https://github.com/TBD54566975/did-dht-method/assets/4887440/a623e7a7-aaa7-48eb-a714-760cb8ce5e32">

Is there a specific reason as to why the root record's `RData` delimits fields using `;` but `_kN` and `_sN` uses `,`? i think we should use `;` to delimit fields so that we can use `,` for multiple values for a given field. 

If not, there are typos here: 

<img width="640" alt="image" src="https://github.com/TBD54566975/did-dht-method/assets/4887440/c7245ea5-f782-4fa2-bb6d-af4983a0ba11">

* `_s0._did.`'s RData value should be `t=LinkedDomains,uri=foo.com,…` not `t=LinkedDomains;uri=foo.com;…`
* `_s1._did` has the same issue

**Comment by thereisnogabe** [comment]
`;` separates distinct entities (like different verification relationships) whereas `,` delineates multiple values for a given field

there is indeed a typo below, which I will address.

**Comment by mistermoe** [comment]
yeah in that case i think `_sN._did.` should absolutely use `;` to separate entities vs. `,` as you have right now because values like `uri` can have multiple values as noted [here](https://www.w3.org/TR/did-core/#dfn-serviceendpoint). 

![image](https://github.com/TBD54566975/did-dht-method/assets/4887440/f1036424-6fce-4066-9023-b32a14234102)


On that note, did `did:dht` make an explicit decision to not support `serviceEndpoint` being an object? DWNs require this which you can see [here](https://identity.foundation/decentralized-web-node/spec/#service-endpoints)

![image](https://github.com/TBD54566975/did-dht-method/assets/4887440/b6c8a517-a2e0-4007-9984-3df899cf8f60)

**Comment by thereisnogabe** [comment]
@mistermoe the spec here creates a profile (method) of DID core. so even though you're correct that DID core supports multiple values, the spec as written does not.

I think we should close this issue (after #84 is merged) and open a new issue to discuss services with multiple values.

**Comment by thereisnogabe** [comment]
fixed in #84

**Comment by mistermoe** [comment]
sounds good!
